### PR TITLE
added python 3.8 in test matrix

### PR DESCRIPTION
**Pull request by iamsobanjaved**
- Added python3.8 travis worker
- Added Django 3.0 in tox file for testing
- Removed support for python < 3.5 and Django < 2.2

https://openedx.atlassian.net/browse/BOM-1578

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def package_data(pkg, roots):
         'sql_grader',
     ],
     install_requires=[
-        'Django',
+        'Django<3.0',
```
Comment (jmbowman) [inline setup.py:36]: Was this needed for tests to pass?  We'll need to remove this fairly soon to start testing against 3.0.

File: tox.ini [hunk 2]
```diff
@@ -2,17 +2,15 @@
 envlist =
     csslint
     eslint
-    py{35}-django{111,20,21,22}
+    py{35,38}-django{22,30}
```
Comment (jmbowman) [inline tox.ini:5]: https://docs.djangoproject.com/en/3.0/releases/3.0/

File: setup.py [hunk 3]
```diff
@@ -26,7 +26,7 @@ def package_data(pkg, roots):
 
 setup(
     name='xblock-sql-grader',
-    version='0.0.1',
+    version='0.0.2',
```
Comment (jmbowman) [inline setup.py:29]: No changes to the package itself were needed, and we're not deploying to PyPI yet, so probably don't need to increment the version.

Comment (iamsobanjaved) [reply setup.py]: Reverted

**Pull request merged**